### ISSUE TITLE
Which version of backbone used in yolov5

### ISSUE THREAD

**Issue by twangnh** [ultralytics/yolov5#548]
Hi thanks for sharing the wonderful work! I'm wondering which CSP backbone is used as from the paper:
http://openaccess.thecvf.com/content_CVPRW_2020/papers/w28/Wang_CSPNet_A_New_Backbone_That_Can_Enhance_Learning_Capability_of_CVPRW_2020_paper.pdf

as I haven't found relevant information in the readme page and code, thank you

**Comment by glenn-jocher** [comment]
@twangnh we created a slightly modified CSP block for YOLOv5:
https://github.com/ultralytics/yolov5/blob/a209a32019da7fee6c4d3209b0950f951c58ec5d/models/common.py#L46-L63

**Comment by bonlime** [comment]
@glenn-jocher 
Could you explain your motivation behind this backbone? How did you come up with it?
I've spent a lot of time trying to understand the idea behind CSPNets and your implementation but I still have questions. 

As far as I understand the core CSP idea is in this piece:
![image](https://user-images.githubusercontent.com/14337581/89453449-f3cb1f00-d767-11ea-8a48-32175fbebd9a.png)

1. Instead of really splitting the input into two parts you use convs `c1` and `c2`. I wonder why not to use `torch.chunk` instead?
2. I suppose this backbone started from Darknet53. How did you come up with number of blocks per stage? Original DarkNet53 had `1, 2, 8, 8, 4`. You have `1, 3, 9, 9, 4`. 

3. Last question is about transition conv `c3` which is "used to truncate the gradients". If I understand correctly you implement the (b) variant of CSP stage. Do you know why according to authors option (c) is faster than (d)? It seems in (d) Transition is applied to a lower number of channels and should be faster

![image](https://user-images.githubusercontent.com/14337581/89455894-b5376380-d76b-11ea-8038-334193e8bf95.png)

**Comment by glenn-jocher** [comment]
@bonlime c1 and c2 operate on the same input, there is no input splitting occurring in our implementation. The blocks per stage are our own making, along with a shared activation and bn layer acting on the c1 and c2 concatenation. Everything else is per the paper, which you may want to consult directly with the author on.

**Comment by bonlime** [comment]
Just to have a reference, I found the answer to the first question here: https://github.com/WongKinYiu/CrossStagePartialNetworks/issues/18

TLDR: splitting and using 2 separate convs implement the same idea.

It still looks to me that splitting should actually be faster. I'll explore it later

**Comment by glenn-jocher** [comment]
@bonlime faster is easy. Faster with the same accuracy though...

**Comment by abhiagwl4262** [comment]
@glenn-jocher Is there a comparative study available between yolov4 & yolov5 ?

**Comment by glenn-jocher** [comment]
@abhiagwl4262 model architectures (including backbones) are defined in each model's yaml configuration file. See https://github.com/ultralytics/yolov5/tree/master/models

**Comment by shashwat1998** [comment]
@glenn-jocher is this backbone CSPResNeXt50 or CSPDarknet53 or any other?
